package.json

```diff
@@ -47,6 +47,7 @@
     "joi": "^14.3.0",
     "joi-date-extensions": "^1.2.0",
     "joi-postcode": "^3.0.0",
+    "jwt-decode": "^2.2.0",
```

Comment:
I realised you can just pull the user's roles out of the JWT rather than making an api call as NN does